presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -706,7 +721,9 @@ private List<DataOutput> bufferFileFooter()
                 closedStripes.stream()
                         .map(ClosedStripe::getStatistics)
                         .map(StripeStatistics::getColumnStatistics)
-                        .collect(toList()));
+                        .collect(toList()),
+                streamSizeHelper.getNodeSizes(),
+                streamSizeHelper.getKeySizes());
```

Comment:
Pass the storage sizes to the ColumnStatistics.merge function used to merge all "stripe" column statistics into a file column statistic. 

Note: there is no such thing as a stripe column statistic in the stripe metadata, it's just a concept used by the column writers.